src/platform_impl/macos/view.rs

```diff
@@ -257,8 +260,16 @@ lazy_static! {
             sel!(acceptsFirstMouse:),
             accepts_first_mouse as extern "C" fn(&Object, Sel, id) -> BOOL,
         );
+        decl.add_method(
+            sel!(clearMarkedText),
+            clear_marked_text as extern "C" fn(&mut Object, Sel),
+        );
```

Comment:
As far as I can tell, it would be simpler if `clear_marked_text` would *not* be a member function. It could instead be a regular rust function, in which case you wouldn't need to add it as a method here, and he function could be defined like this:
```rust
fn clear_marked_text(this: &mut Object) {
    ...
}
```

Reply:
That sounds good !